src/app/legacy/containers/ArticleTimestamp/index.jsx

```diff
@@ -18,9 +17,9 @@ const LastUpdatedTimestamp = Timestamp;
 const ArticleTimestamp = ({
   firstPublished,
   lastPublished,
-  popOut,
-  minutesTolerance,
-  className,
+  popOut = true,
+  minutesTolerance = 0,
+  className = '',
```

Comment:
```suggestion
  className,
```
I can't see `className` in the default props?

Reply:
This needs set to a default value to satisfy the TS components that use this component. It detects that `className` is a prop, but without a default it will complain and ask you to provide a value for every implementation of the component.